src/encoded/static/components/award.js

```diff
@@ -126,11 +158,163 @@ function createDoughnutChart(chartId, values, labels, colors, baseSearchUri, nav
     });
 }
 
+function createBarChart(chartId, unreplicatedLabel, unreplicatedDataset, isogenicDataset, anisogenicDataset, colors, labels, replicatelabels, baseSearchUri, navigate) {
```

Comment:
Given that there are so many parameters, could you add comments to this function like we have for createDoughnutChart? In JSDoc format would be even better, since I’ll be gradually moving us in that direction. See https://github.com/ENCODE-DCC/encoded/blob/ENCD-3439/src/encoded/static/components/globals.js#L118-L128